### PR TITLE
Pull request 

### PR DESCRIPTION
**Pull request by haroonrl**
Dear professor,

Please review and pull if everything is accordingly now. 

Thanks

### CODE REVIEW

File: _data/papers.yml [hunk 1]
```diff
@@ -223,3 +223,13 @@
   year: 2012
 
   copy_publisher: https://www.combinatorics.org/ojs/index.php/eljc/article/view/v19i1p54
+ 
+- title: "A 974GOPS/W Multi-level Parallel Architecture for Binary Weight Network Acceleration"
+  authors: [
+    "haroon.ahmed",
```
Comment (jeehoonkang) [inline _data/papers.yml:229]: Probably it's not a single-author paper. Please describe the other authors (see, e.g., "Derek Dreyer" in this file.)

File: _data/people.yml [hunk 2]
```diff
@@ -460,3 +460,14 @@
...
   ]
+  
+- id: haroon.ahmed
+  name: Haroon Ahmed
+  status: present
+  title: PhD Student
+  place: Rm. 4441, Bldg. E3-1, KAIST (+82-42-350-7878)
+  github: haroonrl
+  educations: [
+      "(2017) M.E Electrical Engineering. Southeast University, China.",
```
Comment (jeehoonkang) [inline _data/people.yml:471]: Please mind the format here. See the other's educations fields.

**Review by jeehoonkang — CHANGES_REQUESTED** [review]
@haroonrl Please don't close a PR and open a new PR. Update this PR and then ping me at cp-chat.kaist.ac.kr .

**Comment by haroonrl** [comment]
Dear Professor,

Please review the recent commit. 

Thanks

**Pull request merged**